scripts/resolve_api_audit.py

```
@@ -0,0 +1,248 @@
+        return "skipped", "no clip on V1 to probe"
+    if not hasattr(item, "ClearClipColor") or not hasattr(item, "SetClipColor"):
+        return "limited", "SetClipColor present but ClearClipColor missing"
+    original = item.GetClipColor() if hasattr(item, "GetClipColor") else None
+    set_ok = item.SetClipColor("Orange")
+    if original:
+        item.SetClipColor(original)
+    else:
+        try:
+            item.ClearClipColor()
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)